config/environments/production.rb

```diff
@@ -19,7 +19,7 @@
   config.public_file_server.headers = {"cache-control" => "public, max-age=#{1.year.to_i}"}
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  config.asset_host = "https://www.rubyvideo.dev"
+  config.asset_host = "https://staging.rubyvideo.dev"
```

Comment:
No you are right I need to change that, it was part of my testing

Reply:
Is this Right?